### ISSUE TITLE
Visual Developer Like NoFlo

### ISSUE THREAD

**Issue by sirinath** [kuujo/vertigo#23]
Is it possible to have a visual developer for Vertigo like NoFlo. Following project might be interesting to impliment this: http://eclipse.org/sirius/

**Comment by arnos** [comment]
Actually Henri Bergius of NoFlo.js mentioned that the NoFlo.js front-end (flowhub.js) will communicate using a common protocol so that different FBP backends can be used (see http://bergie.iki.fi/blog/noflo-update/).

The documentation of the current protocol is at http://noflojs.org/documentation/protocol/.

**Comment by kuujo** [comment]
No-Flo would be great! We'd love tobacco a visual UI for editing Vertigo networks, and actually I've been working pretty diligently on the back end to make runtime network configuration changes a reality. Additionally, support for things like metrics has already been added to the next version. 

It seems that No-Flo's features would integrate seamlessly with those of Vertigo. I know there has been some work on a UI already, and it's definitely one of our primary focuses for the next version along with fault-tolerance, performance, and runtime configuration changes.
